### PR TITLE
Added Marathi (mar) translations

### PR DESCRIPTION
**Pull request by bantya**
Added Marathi (mar) translations #130

### CODE REVIEW

File: resources/lang/mar.ini [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+tool.debug.lore.1 = "माहिती मिळविण्यासाठी ब्लॉक वर डावे क्लिक करा"
+tool.debug.lore.2 = "ब्लॉकचे नाव आणि नुकसान मूल्ये यासारख्या"
+tool.debug.lore.3 = "कार्यक्षमता बदलण्यासाठी //toggledebug वापरा"
+tool.debug.disabled = "डीबग साधन निष्क्रिय केले आहे. ते पुन्हा सक्रिय करण्यासाठी //toggledebug वापरा"
+tool.debug.setenabled = "{%0} आता डीबग साधन आहे!"
+; flood tool
+ui.flood.title = "पूर मेनू"
+ui.flood.options.limit = "जास्तीत जास्त ब्लॉक्स"
+ui.flood.options.blocks = "ब्लॉक्स"
+ui.flood.options.blocks.placeholder = "अर्धविरामांनी विभक्त (वेगळे ) केलेले ब्लॉक्स"
```
Comment (inxomnyaa) [inline resources/lang/mar.ini]: Extra space

File: resources/lang/mar.ini [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+disabled = "निष्क्रिय केले"
+confirmation = "पुष्टीकरण"
+confirmation.yes = "होय"
+confirmation.no = "नाही"
+; errors
+error = "एक त्रुटी आली"
+error.command-error = "कदाचित आपण एक मापदंड (parameter) गमावत आहात किंवा चुकीची कमांड वापरली आहे!"
+error.runingame = "कृपया ही आज्ञा खेळ चालू असताना चालवा!"
+error.limitexceeded = "आपण एकाच वेळी अनेक  ब्लॉक संपादित करण्याचा प्रयत्न करीत आहात. कमी ब्लॉक निवड  किंवा मर्यादा वाढवा"
+error.notarget = "लक्ष्य ब्लॉक आढळला नाही. आवश्यक असल्यास //setrange वापरुन टूल पल्ला (range ) वाढवा"
```
Comment (inxomnyaa) [inline resources/lang/mar.ini]: Extra space

File: resources/lang/mar.ini [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+; commands
+command.info.title = "माहिती"
+command.limit.current = "वर्तमान मर्यादा: {%0}"
+command.limit.set = "ब्लॉक बदल मर्यादा {%0} इतकी सेट केली आहे"
+command.setrange.current = "वर्तमान पल्ला (रेंज): {%0}"
+command.setrange.set = "टुल रेंज सेट {%0} इतकी सेट केली आहे"
+command.biomeinfo.attarget = "बायोम निशाण्यावर आहे"
+command.biomeinfo.atposition = "बायोम ठिकाणावर आहे"
+command.biomeinfo.result = "निवडीत {%0} बायोम सापडले"
+command.biomeinfo.result.line = "ओळख: {%0} नाव : {%1}"
```
Comment (inxomnyaa) [inline resources/lang/mar.ini]: Extra space

File: resources/lang/mar.ini [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+command.biomelist.title = "बायोम यादी"
+command.biomelist.result.line = "ओळख: {%0} नाव : {%1}"
```
Comment (inxomnyaa) [inline resources/lang/mar.ini]: Extra space

File: resources/lang/mar.ini [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+tool.wand.lore.3 = "कार्यक्षमता बदलण्यासाठी //togglewand वापरा"
+tool.wand.disabled = "कांडीचे साधन (wand tool) निष्क्रिय केले. ते पुन्हा कार्यान्वित करण्यासाठी //togglewand वापरा"
+tool.wand.setenabled = "कांडीचे साधन आता {%0} आहे!"
+; debug tool
+tool.debug = "डीबग साधन"
+tool.debug.lore.1 = "माहिती मिळविण्यासाठी ब्लॉक वर डावे क्लिक करा"
+tool.debug.lore.2 = "ब्लॉकचे नाव आणि नुकसान मूल्ये यासारख्या"
+tool.debug.lore.3 = "कार्यक्षमता बदलण्यासाठी //toggledebug वापरा"
+tool.debug.disabled = "डीबग साधन निष्क्रिय केले आहे. ते पुन्हा सक्रिय करण्यासाठी //toggledebug वापरा"
+tool.debug.setenabled = "{%0} आता डीबग साधन आहे!"
```
Comment (inxomnyaa) [inline resources/lang/mar.ini]: Maybe the position of {%0} (which will be "enabled"/"disabled") should be similar to the tool.wand.setenabled entry

File: resources/lang/mar.ini [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+error = "एक त्रुटी आली"
+error.command-error = "कदाचित आपण एक मापदंड (parameter) गमावत आहात किंवा चुकीची कमांड वापरली आहे!"
+error.runingame = "कृपया ही आज्ञा खेळ चालू असताना चालवा!"
+error.limitexceeded = "आपण एकाच वेळी अनेक  ब्लॉक संपादित करण्याचा प्रयत्न करीत आहात. कमी ब्लॉक निवड  किंवा मर्यादा वाढवा"
+error.notarget = "लक्ष्य ब्लॉक आढळला नाही. आवश्यक असल्यास //setrange वापरुन टूल पल्ला (range ) वाढवा"
+error.noselection = "कोणतीही निवड आढळली नाही - प्रथम एक क्षेत्र निवडा"
+error.selectioninvalid = "निवड वैध नाही! सर्व पोझिशन्स सेट केल्या आहेत का ते तपासा!"
+error.nosession = "सत्र (सेशन) तयार केले नाही - कदाचित {%0} वापरण्याची परवानगी नाही"
+error.noclipboard = "कोणताही क्लिपबोर्ड सापडला नाही - प्रथम एक क्लिपबोर्ड तयार करा"
+warning.differentlevel = "[चेतावणी] आपण सध्या ज्या पातळीवर  नाही त्या पातळीवर आपण संपादन करीत आहात!"
```
Comment (inxomnyaa) [inline resources/lang/mar.ini]: Double space near पातळीवर  नाही

**Review by inxomnyaa — CHANGES_REQUESTED** [review]
Could you please remove the marked extra spaces? This is just asthetic, the translation looks done correctly - Thanks for your submission!

**Comment by bantya** [comment]
Fixed the spacing issue. You can check it now. 😀

P.S. Writing Indic languages in Windows is a mess. 😒

File: resources/lang/mar.ini (hunk 6 above)
Comment (bantya) [reply resources/lang/mar.ini]: Done it already..

File: resources/lang/mar.ini (hunk 1 above)
Comment (bantya) [reply resources/lang/mar.ini]: Fixed it.

File: resources/lang/mar.ini (hunk 2 above)
Comment (bantya) [reply resources/lang/mar.ini]: Fixed it.

File: resources/lang/mar.ini (hunk 3 above)
Comment (bantya) [reply resources/lang/mar.ini]: Fixed it.

File: resources/lang/mar.ini (hunk 4 above)
Comment (bantya) [reply resources/lang/mar.ini]: Fixed it.

File: resources/lang/mar.ini (hunk 5 above)
Comment (bantya) [reply resources/lang/mar.ini]: Yes, you are right!
For the first pass I went with literal meaning. Above comment made me understand the the correlation between `enabled/disabled` and `tool.debug.setenabled `. 
Fixed it now.

**Comment by bantya** [comment]
Also, sorry for 3 extra commits. 😏😖

**Comment by inxomnyaa** [comment]
> Also, sorry for 3 extra commits. 😏😖

No need to be sorry, that is what PRs are for - i appreciate any help i get :)

Thanks for fixing

**Review by inxomnyaa — APPROVED** [review]

**Pull request merged**